concepticondata/conceptlists/Constenla-2005-110.tsv

```diff
@@ -0,0 +1,111 @@
+Constenla-2005-110-33	33	FIVE	493	FIVE	CINCO
+Constenla-2005-110-95	95	FLEA	232	FLEA	PIOJO
+Constenla-2005-110-8	8	FLOWER	239	FLOWER	FLOR
+Constenla-2005-110-26	26	FOOT	1301	FOOT	PIE
+Constenla-2005-110-52	52	FOREST	420	FOREST	BOSQUE
+Constenla-2005-110-83	83	FOUR	1500	FOUR	CUATRO
+Constenla-2005-110-73	73	GIVE	1447	GIVE	DAR
+Constenla-2005-110-15	15	GOOD	1035	GOOD	BUENO
+Constenla-2005-110-4	4	GREEN	1425	GREEN	VERDE
+Constenla-2005-110-65	65	GUTS	1334	GUTS	TRIPAS
+Constenla-2005-110-96	96	HAIR	1040	HAIR	CABELLO
+Constenla-2005-110-40	40	HAND	1277	HAND	MANO
+Constenla-2005-110-61	61	HE	1211	HE	ÉL
+Constenla-2005-110-104	104	HEAD	1256	HEAD	CABEZA
+Constenla-2005-110-75	75	HEAR	1408	HEAR	OÍR
+Constenla-2005-110-67	67	HEAVY	1210	HEAVY	PESADO
+Constenla-2005-110-63	63	HIT	1433	HIT	GOLPEAR
+Constenla-2005-110-79	79	HOT OR WARM	2272	HOT OR WARM	CALIENTE
+Constenla-2005-110-11	11	I	1209	I	YO
+Constenla-2005-110-68	68	KILL	1417	KILL	MATAR
+Constenla-2005-110-10	10	KNOW	3626	KNOW	SABER
+Constenla-2005-110-97	97	LAUGH	1355	LAUGH	REÍR
+Constenla-2005-110-102	102	LEAF	628	LEAF	HOJA
+Constenla-2005-110-20	20	LIVER	1224	LIVER	HÍGADO
+Constenla-2005-110-91	91	MAN	1554	MAN	HOMBRE
+Constenla-2005-110-5	5	MEAT	634	MEAT	CARNE
+Constenla-2005-110-24	24	MOON	1313	MOON	LUNA
+Constenla-2005-110-74	74	MOTHER	1216	MOTHER	MADRE
+Constenla-2005-110-17	17	MOUNTAIN	639	MOUNTAIN	MONTAÑA
+Constenla-2005-110-22	22	MOUTH	674	MOUTH	BOCA
+Constenla-2005-110-84	84	NECK	1333	NECK	CUELLO
+Constenla-2005-110-78	78	NEW	1231	NEW	NUEVO
+Constenla-2005-110-80	80	NIGHT	1233	NIGHT	NOCHE
+Constenla-2005-110-35	35	NO	1269	NO	NO
+Constenla-2005-110-58	58	NOSE	1221	NOSE	NARIZ
+Constenla-2005-110-21	21	OLD	1229	OLD	VIEJO
+Constenla-2005-110-44	44	ONE	1493	ONE	UNO
+Constenla-2005-110-87	87	PATH	2252	PATH	CAMINO
+Constenla-2005-110-16	16	RED	156	RED	ROJO
+Constenla-2005-110-43	43	SALT	1274	SALT	SAL
+Constenla-2005-110-1	1	SAND	671	SAND	ARENA
+Constenla-2005-110-82	82	SAY	1458	SAY	DECIR
+Constenla-2005-110-103	103	SEE	1409	SEE	VER
+Constenla-2005-110-34	34	SING	1261	SING	CANTAR
+Constenla-2005-110-88	88	SISTER	1263	SISTER	HERMANA
+Constenla-2005-110-14	14	SLEEP	1585	SLEEP	DORMIR
+Constenla-2005-110-106	106	SMALL	1246	SMALL	PEQUEÑO
+Constenla-2005-110-56	56	SMELL	2124	SMELL	OLER
+Constenla-2005-110-7	7	SMOKE (EXHAUST)	778	SMOKE (EXHAUST)	HUMO
+Constenla-2005-110-99	99	SNAKE	730	SNAKE	SERPIENTE
+Constenla-2005-110-92	92	STOMACH	819	STOMACH	VIENTRE
+Constenla-2005-110-13	13	STONE	857	STONE	PIEDRA
+Constenla-2005-110-90	90	SUN	1343	SUN	SOL
+Constenla-2005-110-81	81	TAIL	1220	TAIL	COLA
+Constenla-2005-110-12	12	TALK	2708	TALK	HABLAR
+Constenla-2005-110-98	98	THIS	1214	THIS	ESTE
+Constenla-2005-110-23	23	THOU	1215	THOU	TÚ
+Constenla-2005-110-30	30	THREE	492	THREE	TRES
+Constenla-2005-110-108	108	TONGUE	1205	TONGUE	LENGUA
+Constenla-2005-110-25	25	TOOTH	1380	TOOTH	DIENTE
+Constenla-2005-110-107	107	TREE	906	TREE	ÁRBOL
+Constenla-2005-110-27	27	TUMBLE (FALL DOWN)	2894	TUMBLE (FALL DOWN)	CAER
```

Comment:
Considering the English gloss, this is probably mapped correctly, but all other instances of caer in the database seem to be mapped to FALL instead.